ui/app/src/components/slippagePanel/Index.vue

```diff
@@ -55,18 +55,31 @@ export default defineComponent({
       get: () => props.slippage,
       set: (amount) => handleUpdateSlippage(amount),
     });
+
     function handleUpdateSlippage(amount: string) {
       context.emit("update:slippage", amount);
     }
+
+    function isNumeric(s: any) {
+      return (s - 0) == s && (''+s).trim().length > 0;
```

Comment:
would `!isNaN(s)` work or are there other possibilities you are thinking about?

Reply:
@ryardley `!isNan()` does not work in some cases. This might look like overkill since we also validate the swap button, but better to be more cautions I think